cldr-json/cldr-dates-full/main/en-GS/ca-gregorian.json

```diff
@@ -329,20 +329,20 @@
               "short": "yMMdd"
             },
             "timeFormats": {
-              "full": "h:mm:ss a zzzz",
+              "full": "HH:mm:ss zzzz",
               "full-alt-ascii": "h:mm:ss a zzzz",
-              "long": "h:mm:ss a z",
+              "long": "HH:mm:ss z",
               "long-alt-ascii": "h:mm:ss a z",
-              "medium": "h:mm:ss a",
+              "medium": "HH:mm:ss",
               "medium-alt-ascii": "h:mm:ss a",
-              "short": "h:mm a",
+              "short": "HH:mm",
               "short-alt-ascii": "h:mm a"
             },
             "timeSkeletons": {
-              "full": "ahmmsszzzz",
-              "long": "ahmmssz",
-              "medium": "ahmmss",
-              "short": "ahmm"
+              "full": "HHmmsszzzz",
+              "long": "HHmmssz",
+              "medium": "HHmmss",
+              "short": "HHmm"
```

Comment:
I'm a little surprised at how many changes there are from alpha1 to alpha2. These changes to en-GS appear to be from https://github.com/unicode-org/cldr/pull/4320 last week.